### ISSUE TITLE
install issue 

### ISSUE THREAD

**Issue by Thexiyang** [cultivarium/GenomeSPOT#4]
thanks for this nice tool. when i install using this: `pip install .`, the issue pops out: 

```
Building wheels for collected packages: genome_spot
  Building wheel for genome_spot (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [34 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib
      creating build/lib/genome_spot
      copying genome_spot/genome_spot.py -> build/lib/genome_spot
      copying genome_spot/helpers.py -> build/lib/genome_spot
      copying genome_spot/_version.py -> build/lib/genome_spot
      copying genome_spot/join_outputs.py -> build/lib/genome_spot
      copying genome_spot/__init__.py -> build/lib/genome_spot
      creating build/lib/genome_spot/bioinformatics
      copying genome_spot/bioinformatics/signal_peptide.py -> build/lib/genome_spot/bioinformatics
      copying genome_spot/bioinformatics/protein.py -> build/lib/genome_spot/bioinformatics
      copying genome_spot/bioinformatics/__init__.py -> build/lib/genome_spot/bioinformatics
      copying genome_spot/bioinformatics/dna.py -> build/lib/genome_spot/bioinformatics
      copying genome_spot/bioinformatics/genome.py -> build/lib/genome_spot/bioinformatics
      creating build/lib/genome_spot/model_training
      copying genome_spot/model_training/download_trait_data.py -> build/lib/genome_spot/model_training
      copying genome_spot/model_training/make_holdout_sets.py -> build/lib/genome_spot/model_training
      copying genome_spot/model_training/run_model_selection.py -> build/lib/genome_spot/model_training
      copying genome_spot/model_training/train_models.py -> build/lib/genome_spot/model_training
      copying genome_spot/model_training/__init__.py -> build/lib/genome_spot/model_training
      copying genome_spot/model_training/make_training_dataset.py -> build/lib/genome_spot/model_training
      creating build/lib/genome_spot/taxonomy
      copying genome_spot/taxonomy/balance.py -> build/lib/genome_spot/taxonomy
      copying genome_spot/taxonomy/taxonomy.py -> build/lib/genome_spot/taxonomy
      copying genome_spot/taxonomy/__init__.py -> build/lib/genome_spot/taxonomy
      copying genome_spot/taxonomy/partition.py -> build/lib/genome_spot/taxonomy
      creating build/lib/genome_spot/bioinformatics/hmm
      copying genome_spot/bioinformatics/hmm/hmm_signal_peptide.joblib -> build/lib/genome_spot/bioinformatics/hmm
      running build_scripts
      creating build/scripts-3.8
      error: [Errno 21] Is a directory: 'genome_spot/'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for genome_spot
  Running setup.py clean for genome_spot
Failed to build genome_spot
ERROR: Could not build wheels for genome_spot, which is required to install pyproject.toml-based projects

```

**Comment by till-bornemann** [comment]
Hey, 
just wanted to mention that i have the same issue running `pip install .` (and terminating with the same error) as Thexiyang.
I'm working on an Ubuntu server, have loaded a conda env made specifically for genomespot and (obviously run the  `pip install . `command in the folder downloaded via your git clone command). I was able to run the `pip install -r requirements.txt` command without errors (i tried that after the `pip install . `failed to see if that might fix the first error).

Additionally, i'd suggest adding the specific python version your software needs (3.8.16 is what an incompatibility error gave me so i changed my python3 version in the conda env to it) in the README dependency list. 

Best,
Till

**Comment by tylerbarnum** [comment]
Thanks for raising these issues. Hopefully resolved by PR #5 (https://github.com/cultivarium/GenomeSPOT/pull/5) so try and again let us know.

The issue causing `error: [Errno 21] Is a directory: 'genome_spot/'` was that I pointed to a directory instead of a script in setup.py. The python version requirement has been relaxed as well. Works for 3.10 and 3.11 in our hands.

**Comment by till-bornemann** [comment]
Hi Tyler,
just wanted to let you know that it indeed fixed it for me.
Thanks!

Till
